core/src/test/scala/weaponregex/parser/ParserTest.scala

```diff
@@ -672,6 +672,24 @@ trait ParserTest {
     parseErrorTest(pattern)
   }
 
+  test("Parse non-hexadecimal value \\xGG") {
+    val pattern = "\\xGG"
+    val parsedTree = Parser(pattern, parserFlavor).get.to[Concat]
+
+    assertEquals(parsedTree.children.length, 3)
+
+    treeBuildTest(parsedTree, pattern)
+  }
+
+  test("Parse non-unicode value \\uGGGG") {
+    val pattern = "\\uGGGG"
+    val parsedTree = Parser(pattern, parserFlavor).get.to[Concat]
+
+    assertEquals(parsedTree.children.length, 5)
+
+    treeBuildTest(parsedTree, pattern)
+  }
+
```

Comment:
These should invalid in JVM... but they don't 🤦‍♂️ I will need to look at their parsing rules again.

May I use your changes & tests on my branch and close this PR?